components/Releases/UpdateRelease.jsx

```diff
@@ -24,6 +23,15 @@ export default function UpdateRelease({
     release.is_password_protected
   )
   const [pagePassword, setPagePassword] = useState(release.page_password)
```

Comment:
Wondering if we could simplify this naming to just `password` instead of `pagePassword`. What do you think?